array_test.go

```diff
@@ -666,7 +682,11 @@ func TestArray_IsEqualUnordered(t *testing.T) {
 		value.chain.assertFailed(t)
 		value.chain.clearFailed()
 
-		value.NotEqualUnordered(func() {})
+		value.IsEqualUnordered("123")
+		value.chain.assertFailed(t)
+		value.chain.clearFailed()
+
+		value.NotEqualUnordered("123")
 		value.chain.assertFailed(t)
 		value.chain.clearFailed()
```

Comment:
Which branches are we covering with these two blocks? Aren't they already covered with:

```go
		value.IsEqualUnordered(func() {})
		value.chain.assertFailed(t)
		value.chain.clearFailed()

		value.NotEqualUnordered(func() {})
		value.chain.assertFailed(t)
		value.chain.clearFailed()
```

?

Reply:
Removed these 2 cases